Add Garden Seeds main page and go-seed (#4693)
garden-io/garden: docs/tutorials/garden-recipes/README.md

---
title: Garden Recipes
order: 2
---

> Welcome to Garden Recipes🍳!

Reviewer: ```suggestion
```

Reviewer: I would remove this as it is just repeating the section title.

This section walks you through using Garden with various development tools and programming languages. Our main focus with this section is to provide practical examples and guides that help you effectively leverage the power of Garden. 🚀

> Following our Garden Recipes, you'll learn how to streamline your development process, improve efficiency, and quickly build robust applications ✨. Whether working on personal projects or collaborating with a team, Garden empowers you to focus on what matters most: creating fantastic software 💡.

Reviewer: ```suggestion
Following our Garden Recipes, you'll learn how to streamline your development process, improve efficiency, and quickly build robust applications ✨.
```

I'm a fan of going to value as quickly as possible.

## Languages

In this [section](./languages/README.md), you'll find various examples and guides tailored to specific coding languages. Whether you're a Pythonista, a Gopher, or working with other programming languages, we've got you covered!

These recipes offer clear explanations and code samples to help you integrate Garden into your preferred language effortlessly.

Each recipe follows a consistent structure, making it a breeze to follow along and apply the concepts to your projects ✅.

> ## Tooling

Reviewer: Should this section instead be called something like Integrations? @eysi09

Reviewer 2: I would avoid to call it integrations, since could be confused with Cloud Integrations/connections. Maybe we can even be more specific and call it CI/CD tools? Or something along those lines.

> This section provides examples and guides for integrating Garden with popular development tools and platforms🤖.

Reviewer: ```suggestion
This section provides examples and guides for integrating Garden with popular development tools and platforms 🤖.
```

Added space between word and emoji.

From GitHub Actions to GitLab CI, we cover a range of tooling options to help you automate your workflows, deploy your applications, and optimize your environments.

## Contribute

> We value contributions from our community to enhance the Garden Recipes collection, we encourage you to get involved by providing feedback, feature requests, and bug reports.

Reviewer: ```suggestion
```

Next two paragraphs are a bit verbose. I'd remove in favor of getting to how they contribute as you do with the third paragraph.

> Your input is crucial in shaping the future of Garden Recipes and ensuring its relevance and usefulness to the community.

Reviewer: ```suggestion
```

See previous comment.

If you have an idea for a new recipe or would like to see improvements to existing ones, you can use our feature request template to share your suggestions.

> If you encounter any issues or bugs in the recipes, please report them in our main repository. We can create a rich and valuable resource for the Garden community.

Reviewer: ```suggestion
If you encounter any issues or bugs in the recipes, please report them in our main repository. Or [join our Discord](https://go.garden.io/discord) to talk to our devs or get help.
```
